src/main/res/layout/activity_microphone_configure.xml

```diff
@@ -1,5 +1,4 @@
-
-      <?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@lukeswitz is it better?

Reply:
Negative, the changes here: 
![screen shot 2018-04-21 at 11 29 20 am](https://user-images.githubusercontent.com/10099969/39087028-5b35107e-4557-11e8-85d8-b9ca449b024c.png)
indicate the opposite. I am not the place to learn proper PR's, please reference this in the meantime: http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/ 

Best,
Luke

Reply:
thank you, so much. sorry if i do something wrong. i'm just a beginner and there are a lot of new for me. thank you for patience

Reply:
Nice work cleaning this up. I’d remove the string value change however as it is fine as it is (perhaps not localized in your language; if you want, add them to your PR and switch the English back).